### PR TITLE
02.Git.Local

### PR DESCRIPTION
**Pull request by Zhenyakuch**

### CODE REVIEW

File: Evgeniya_Pehota/02.Git.Local/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+  494  cd C:\academy.by
```
Comment (pluhin) [inline Evgeniya_Pehota/02.Git.Local/README.md:1]: Please use git markdown, for your report

File: Evgeniya_Pehota/02.Git.Local/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  536  git log --oneline
+  537  git checkout dev
+  538  ~git push origin master
+  539  git push origin master
+  540  git push origin master
+  541  git push origin
+  542  git push origin --all
+  543  git history
+  544  git -history
+  545  history
```
Comment (pluhin) [inline Evgeniya_Pehota/02.Git.Local/README.md:52]: Missing list of commits for each branches in report

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by pluhin** [comment]
Please, change name of PR, the mean `№` need to replace

**Comment by pluhin** [comment]
Please squash commits to one or two commits

**Comment by Zhenyakuch** [comment]
/

**Pull request closed without merging**